src/NPFinancial.jl

```diff
@@ -288,5 +288,19 @@ function mirr(values::AbstractVector{<:Real}, finance_rate::Real, reinvest_rate:
     return (numer/denom)^(1/(n - 1))*(1 + reinvest_rate) - 1
 end
 
+"""
+    eir(rate::Real, nper::Real)
+
+Computes the effective interest rate `ieff` given 
+* `rate` is the nominal (annual) rate,
+* `nper` is the number of compounding periods.
+"""
+
+function eir(rate::Real, nper::Real)
+    ieff = (((1 + (rate / nper))^nper)-1)
+    
+    return ieff
```

Comment:
let's just return the expression directly rather than assigning variable and returning it.